Sources/NIOCore/Channel.swift

```diff
@@ -378,7 +378,7 @@ extension ChannelError: Equatable { }
 /// The removal of a `ChannelHandler` using `ChannelPipeline.removeHandler` has been attempted more than once.
 public struct NIOAttemptedToRemoveHandlerMultipleTimesError: Error {}
 
-public enum DatagramChannelError {
+public enum DatagramChannelError: Sendable {
```

Comment:
This is just a namespace right? We probably shouldn't make it `Sendable` right now

Reply:
Okay, I will mark it then as non-sendable. I don't think it really matters though.